include/vcpkg/base/messages.h

```diff
@@ -554,6 +554,11 @@ namespace vcpkg
                     (msg::spec),
                     "'HEAD' means the most recent version of source code",
                     "{spec} is already installed -- not building from HEAD");
+    DECLARE_MESSAGE(AmbiguousConfigDeleteConfigFile,
+                    (msg::path),
+                    "",
+                    "Ambiguous vcpkg configuration provided by both manifest and configuration file.\n-- Delete "
+                    "configuration file \"{path}/vcpkg-configuration.json\"");
```

Comment:
The other alternative (removing the manifest file) should be part of the same message. Also, I think adding `/vcpkg-configuration.json` should be done as it was before with `Path::operator/` rather than trying to do it in the localized message, so that it has a `\` on Windows.

Reply:
I had to split the messages because they included two different paths, and I can't reuse the msg::path replacement.